src/Lykke.Service.KucoinAdapter/Modules/ServiceModule.cs

```diff
@@ -46,6 +47,13 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<ShutdownManager>()
                 .As<IShutdownManager>();
 
+            builder.RegisterType<OrderbookPublishingService>()
+                .WithParameter("orderbookSettings", _settings.CurrentValue.Orderbooks)
+                .WithParameter("rabbitMqSettings", _settings.CurrentValue.RabbitMq)
+                .As<IStopable>()
+                .As<IStartable>()
```

Comment:
Do not use IStartable feature, StartapManager should start its dependencies. https://github.com/LykkeCity/lykke.dotnettemplates/blob/8958ac68b8eaa3e1e2907b2cb725dd7aed03d9f0/Lykke.Job.LykkeJob/src/Lykke.Job.LykkeJob.Services/StartupManager.cs#L14